### DISCUSSION TITLE
Automatically re-share notes that have linked to a note which is now itself shared (e.g. now has a public share link)

### DISCUSSION THREAD

**Discussion by AbdelrhmanUZaki** [alangrainger/share-note#55] [Vote for upcoming features]
Apply the modification and publish it automatically without the need to press the publish button, as in the case of pages that are within each other, you must publish the new page, and publish the original page again because you added a link to the new page in it - meaning that you modified it and this requires re-publishing.

**Comment by aiproductguy** [comment]
By "pages that are within each other" do you mean ![[transclusions]] or just [[wiki-links]]?

How many levels deep should the recursion go?
